volatility/framework/plugins/windows/cmdline.py

```diff
@@ -30,35 +30,54 @@ def get_requirements(cls) -> List[interfaces.configuration.RequirementInterface]
                                          optional = True)
         ]
 
+    @classmethod
+    def get_cmdline(self, context: interfaces.context.ContextInterface, 
+                    kernel_table_name: str, proc):
+        """Extracts the cmdline from PEB
+
+        Args:
+            context: the context to operate upon
+            kernel_table_name: the name for the symbol table containing the kernel's symbols
+            proc: the process object
+
+        Returns:
+            A string with the command line
+        """
+        try:
+            proc_id = proc.UniqueProcessId
+            proc_layer_name = proc.add_process_layer()
+
+        except exceptions.InvalidAddressException as excp:
+            vollog.debug("Process {}: invalid address {} in layer {}".format(proc_id, excp.invalid_address,
+                                                                             excp.layer_name))
+            return "Invalid address for PID"
```

Comment:
It's difficult to distinguish this from having had this typed in the commandline.  Ideally the classmethod would throw the exception and the generator (whatever called it) would do the catching and provide the appropriate response.  In the previous plugin, the generator was what was interpreting the exceptions.

Reply:
ok, thanks for the suggestions. I did another commit with the modification. For this specific part I followed how it has been done in procdump.py